lib/puppet/provider/network_config/redhat.rb

```diff
@@ -181,7 +181,7 @@ def self.format_file(filename, providers)
     props    = {}
 
     # Map everything to a flat hash
```

Comment:
I think so, yes, but it seems to refer to the block below the assignment to props. I'll just move it if that's the case